webplugin/template/mck-sidebox.html

```
@@ -1727,31 +1664,18 @@ <h3 id="km-csat-feedback-heading">Add feedback (optional)</h3>
                         </svg>
                     </button>
                 </div>
-                <div class="mck-voice-interface-body">
+                <div>
                     <div class="mck-voice-image-container">
                         <div class="voice-ring voice-ring-1"></div>
                         <div class="voice-ring voice-ring-2"></div>
                         <div class="voice-ring voice-ring-3"></div>
                         <img class="mck-voice-image" src="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)